lessons/Assignment-3-Ruby-Part-1.md

```diff
@@ -32,76 +35,116 @@ if (x > 18) {
     console.log("x is less than or equal to 11")
 }
 ```
+
 and here is Ruby:
-```
+
+```ruby
 x = 17
 if x > 18
   puts "x is more than 18"
 elsif x > 11
-  puts "x is more than 11" 
+  puts "x is more than 11"
 else
   puts "x is less than or equal to 11"
 end
 ```
+
 As you can see, there are some syntactic differences.  For example, there is the new word "elsif", and curly braces are used less often, except in hashes.
 
+Ruby has an additional keyword, `unless`, which is effectively `if not`, and it can be paired with `else` in the same way that if does.
+
+```ruby
+x = 0
+
+unless x.positive?
+  warn "x is not a positive number"
+end
+```
+
+In Ruby, the `if`/`unless` can also come after an expression at the end of a line, but with this syntax you can't use `elsif` or `else` with it. For example:
+
+```ruby
+x = 0
+warn "x is not a positive number" unless x.positive?
+```
+
 ## Loops in Ruby
 
 There are various ways to do loops in Ruby:
-```
-3.times do
+
+```ruby
+5.times do
```

Comment:
I deliberately made most of the loops iterate 5 times so that it would be easier for students to compare them.